include_core/AtomicSupport.hpp

```diff
@@ -108,25 +60,15 @@
  * For AIX, dropSMT should drop to LOW(2) and 
  * restoreSMT should raise back to MEDIUM(4)
  */
-#if defined(__xlC__) && !defined(__GNUC__)
-		inline void __dropSMT() { __ppc_dropSMT(); }
-		inline void __restoreSMT() { __ppc_restoreSMT(); }
-#else
-		inline void __dropSMT() {  __asm__ volatile ("or 1,1,1"); }
-		inline void __restoreSMT() {  __asm__ volatile ("or 2,2,2"); }
-#endif
+		inline void __dropSMT() { __asm volatile ("or 1,1,1"); }
```

Comment:
Should be `__asm__` here, I'm not sure `__asm` will be recognized, according to GCC's docs anyway.

Also we have `volatile` and `__volatile__`; it would be better to settle on `__volatile__` throughout to match `__asm__` for consistency.

Reply:
Good idea.  Updated.